Sources/Xcodeproj/pbxproj().swift

```diff
@@ -415,6 +415,14 @@ func xcodeProject(
                        + "ln -sf \"${BUILT_PRODUCTS_DIR}/${EXECUTABLE_PATH}\" \"${PROJECT_TEMP_DIR}/SymlinkLibs/lib${EXECUTABLE_NAME}.dylib\"\n"
             target.addShellScriptBuildPhase(script: script)
         }
+
+        // Hack: If we're bootstrapping add this shell script phase for package description
+        // target. This will let us log the build time TOOLCHAINS value and then read it later
+        // when running tests. This means we can change the toolchain from Xcode preferences
+        // and run tests without doing anything else.
+        if getenv("SWIFTPM_BOOTSTRAP") != nil && module.name == "PackageDescription" {
+            target.addShellScriptBuildPhase(script: "echo $TOOLCHAINS > ${TARGET_BUILD_DIR}/toolchains-build-time-value.log")
+        }
```

Comment:
Yeah, this is horrible.  Is there really no way to get this from Xcode when running the tests?

Also, if it has to be here, then it seems that `pbxproj().swift` itself should have some more general facility and the hack should be in the code that calls it.  I know it's a hack but having a direct module name comparison in the middle of general-purpose code is too much to stomach.